FEATURE-LIST.md

```diff
@@ -0,0 +1,255 @@
+# Git Machete Feature List
+
+## Table of Contents
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+<!-- To install doctoc run `npm install -g doctoc`, to use it run `doctoc <this-file-path> -->
+
+- [Where to find the plugin tab](#where-to-find-the-plugin-tab)
+- [Branch graph](#branch-graph)
+- [Check out branches](#check-out-branches)
+- [Toggle listing commits](#toggle-listing-commits)
+- [Rebase](#rebase)
+- [Push](#push)
+- [Pull](#pull)
+- [Reset to remote](#reset-to-remote)
+- [Fast forward parent](#fast-forward-parent)
+- [Slide out branch](#slide-out-branch)
+- [Slide in branch](#slide-in-branch)
+- [Override fork point](#override-fork-point)
+- [Discover](#discover)
+- [Edit machete file](#edit-machete-file)
+- [Other actions](#other-actions)
+- [Multi-repository support](#multi-repository-support)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+## Where to find the plugin tab
+
+Git Machete IntelliJ Plugin is available under the `Git` tool window in the `Git Machete` tab.
+You can also use `Ctrl` + `Alt` + `Shift` + `M` shortcut to open it.
+
+![](docs/open_git_machete.gif)
+
+
+## Branch graph
+
+For each branch, Git Machete indicates the relation to each of its child branches.
+If the edge between them is **green** that means the child branch is in sync with its parent branch &mdash; in other words, there are no commits in the parent branch that don't belong to the child.
+But if there are some commits in the parent branch that are **not** reachable from the child, then the edge is **red** &mdash; you need to [rebase](#rebase) child branch onto the parent.
+Edge can also be **gray** or **yellow** &mdash; that means respectively the branch was merged to the parent, or a fork point can't be determined automatically
```

Comment:
Pls keep up to date with the updated wording in the base PR (separate sentences for `gray` and `yellow`)